StratoSnap/CameraController.swift

```diff
@@ -61,14 +65,62 @@ extension CameraController: AVCapturePhotoCaptureDelegate {
     }
 }
 
+
+
+// Camera Settings & Orientation
+extension CameraController {
+    func exifOrientationFromDeviceOrientation() -> UInt32 {
+        enum DeviceOrientation: UInt32 {
+            case top0ColLeft = 1
+            case top0ColRight = 2
+            case bottom0ColRight = 3
+            case bottom0ColLeft = 4
+            case left0ColTop = 5
+            case right0ColTop = 6
+            case right0ColBottom = 7
+            case left0ColBottom = 8
+        }
+        var exifOrientation: DeviceOrientation
+        // assuming back camera always
+        switch UIDevice.current.orientation {
+        case .portraitUpsideDown:
+            exifOrientation = .left0ColBottom
+        case .landscapeLeft:
+            exifOrientation =  .top0ColLeft
+        case .landscapeRight:
+            exifOrientation = .bottom0ColRight
+        default:
+            exifOrientation = .right0ColTop
+        }
+        return exifOrientation.rawValue
+    }
+}
+
 extension CameraController: AVCaptureVideoDataOutputSampleBufferDelegate {
     func captureOutput(_ output: AVCaptureOutput, didOutput sampleBuffer: CMSampleBuffer, from connection: AVCaptureConnection) {
         // TODO: FIX THIS LOGIC UIImage is not being created properly!!
         DispatchQueue.main.async { [unowned self] in
             let uiImage = self.getImageFromSampleBuffer(sampleBuffer: sampleBuffer)
-            print("esketit")
-            self.delegate?.captured(image: uiImage!)
+            //self.delegate.captured(image: uiImage!)
```

Comment:
ok to get rid of this? @AbdulTheProgrammer

Reply:
yeah, you aren't doing anything in the delegate function